discourse_migration/userguide.md

```diff
@@ -45,48 +45,49 @@ are the typical use cases:
 
 This table explains the mapping from mailing lists to categories in Discourse.
 The email addresses of these categories will remain the same, after the
+migration.  Deprecated lists will become read-only as part of the Discourse
```

Comment:
"Deprecated" implies that they will be replaced with some new thing, and are limping along until that happens.

How about "Obsolete" or "Frozen" as names?